app/uk/gov/hmrc/customs/notification/repo/NotificationWorkItemRepo.scala

```diff
@@ -211,9 +212,9 @@ extends WorkItemRepository[NotificationWorkItem, BSONObjectID] (
     val selector = Json.obj(workItemFields.id -> Json.toJsFieldJsValueWrapper(id))
     val update = Json.obj("$inc" -> Json.obj(workItemFields.failureCount -> 1))
 
-    collection.findAndUpdate(selector, update).map(_ => ())
+    findAndUpdate(selector, update, false).map(_ => ())
```

Comment:
i prefer naming boolean parameters, like the method above

Reply:
Ok, will change it.